### PR TITLE
doc: FAQ English version & README update

### PR DESCRIPTION
**Pull request by JoeZijunZhou**
* added `FAQ_EN.md`, author: Lili, proofreader: @JoeZijunZhou
* updated `README_EN.md`, author: @CNYoki , proofreader: @Atena1118 @JoeZijunZhou

### CODE REVIEW

**Comment by JoeZijunZhou** [comment]
@frank-zsy plz approve but do not merge until we get all the docs committed, thanks!

**Comment by frank-zsy** [comment]
> @frank-zsy plz approve but do not merge until we get all the docs committed, thanks!

You can tell me when you finish the PR, the robot will auto merge this PR after I approve

File: README_EN.md [hunk 1]
```diff
@@ -1,48 +1,48 @@
 # Information Collection Platform for Wuhan 2019-nCoV Epidemic Prevention
 
 For the Pneumonia Caused by the Novel Coronavirus (2019-nCoV) broke out in early 2020 in Wuhan, Hubei, China, the project aims to collect information of hospitals, hotels, factories, logistics, donations, contributions, prevention, treatment, and other dynamic information regarding national epidemic prevention, all of this will be collected to release to public 
-We designed to enable information exchange between all parties about the epidemic and allocate social resources effectively. 
+We designed to enable information exchange between all parties about the epidemic and effective social resources allocation .
```
Comment (Atena1118) [inline README_EN.md]: between all parties就够, 个人觉得不需要 about the epidemic . 于是整个这句话就是 ： We designed to enable information exchange between all parties and the effective social resources allocation

File: README_EN.md [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
-- Code warehouse：https://github.com/wuhan2020/map-viz
+- Code repository：https://github.com/wuhan2020/map-viz
 
 ### Online Form Editing Service(ShiMo) Synchronization Component
 
-- Code warehouse：https://github.com/wuhan2020/shimo-sheet2json
+- Code repository：https://github.com/wuhan2020/shimo-sheet2json
 
-## Data Submission
+## Data Submission
```
Comment (Atena1118) [inline README_EN.md]: submission 应该改成 submitting

File: README_EN.md (hunk 1 above)
Comment (JoeZijunZhou) [reply README_EN.md]: cool. 刚刚忘记删除了，因为直接从你的doc copy的。

File: README_EN.md (hunk 2 above)
Comment (JoeZijunZhou) [reply README_EN.md]: 这里submission作名词，其实没有问题，submitting强调动作。哈哈相信我^_^

Comment (Atena1118) [reply README_EN.md]: 好的 好的 哈哈

**Comment by JoeZijunZhou** [comment]
@frank-zsy ready to merge, thanks!

**Pull request merged**